### ISSUE TITLE
Variable name fullInstance doent exist

### ISSUE THREAD

**Issue by Fabian1976** [MantasTumenas/Zabbix-template-for-Microsoft-SQL-Server#32]
https://github.com/MantasTumenas/Zabbix-template-for-Microsoft-SQL-Server/blob/fd95591d5b16c99c8bec8e65d7d36739f4696e05/With%20SQL%20instance%20discovery/Scripts/Discovery.mssql.databasestatus.ps1#L62

It should be `$fullInstanceName`.

Error message is:
```
Error executing query on !
```

And should be:
```
Error executing query on <INSTANCENAME>!
```

**Comment by MantasTumenas** [comment]
You are right. The error handling is not polished as it could be. I will look into it. Will check all scripts.
